vendor-bin/phpunit/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "phpunit/phpunit": "^7.0"
+    },
+    "config": {
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4.0"
```

Comment:
I guess this isn't totally relevant to this PR since you already do this in master, but I don't get why are you guys using this. Let composer install stuff based on php version it really runs on?

Reply:
This has to do with the fact that we commit `composer.lock` too. I'm running php 6 locally (or maybe not, doesn't matter), but I want to generate lock files that will result in a working CI.

Reply:
That said, it should read 7.1.0 here, I'm going to change that